example_project/outer_rim_logistics/docker/postgres-exporter-entrypoint.sh

```
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
+set -eu
+
+password_file="/run/secrets/postgres_password"
+if [ ! -f "$password_file" ]; then
+  echo "Missing postgres password secret at $password_file" >&2
+  exit 1
+fi
+
+export DATA_SOURCE_NAME="postgresql://${POSTGRES_USER:-orl}:$(cat "$password_file")@db:5432/${POSTGRES_DB:-orl}?sslmode=disable"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)